collection_data.json

```diff
@@ -0,0 +1 @@
+[{"model": "api.user", "pk": 1, "fields": {"password": "!GtvpcXzF8mZqjnrL5SA2SF8AwI2vzUeo1Pxmg1D3", "last_login": null, "is_superuser": false, "username": "AnonymousUser", "first_name": "", "last_name": "", "email": "", "is_staff": false, "is_active": true, "date_joined": "2017-09-13T15:21:14.069Z", "gravatar": "", "groups": [], "user_permissions": []}}, {"model": "api.user", "pk": 2, "fields": {"password": "pbkdf2_sha256$30000$7WNyj0rFvFXZ$bDfHZ3f8eO2iWtQmuUUjRtYmuefK96QufoibTFbS82Y=", "last_login": "2017-09-25T15:16:56.764Z", "is_superuser": true, "username": "admin", "first_name": "", "last_name": "", "email": "", "is_staff": true, "is_active": true, "date_joined": "2017-09-13T15:21:50.897Z", "gravatar": "", "groups": [], "user_permissions": []}}, {"model": "api.user", "pk": 3, "fields": {"password": "!DC6SuMOrN2N1T6eLqK7g40UMgSqWWMRRfaC0zXk6", "last_login": "2017-09-25T15:00:53.911Z", "is_superuser": false, "username": "ac2gj", "first_name": "Joshua", "last_name": "Bird", "email": "", "is_staff": false, "is_active": true, "date_joined": "2017-09-13T18:14:32.778Z", "gravatar": "https://secure.gravatar.com/avatar/5a836e6ff8717dba1af6cfca42a59e91?d=identicon", "groups": [], "user_permissions": []}}
```

Comment:
We should separate the api from the collections models, giving it its own app so db dumple arent wierd like this.